hack/lib/golang.sh

```diff
@@ -28,6 +28,7 @@ readonly KUBE_SERVER_TARGETS=(
   cmd/kubernetes
   plugin/cmd/kube-scheduler
 )
+
```

Comment:
the extra line helps readability. Do you want me to create a separate PR to add a new line ?